lib/dict.c

```diff
@@ -254,15 +240,15 @@ int rc_read_dictionary (rc_handle *rh, char const *filename)
 			dval->next = rh->dictionary_values;
 			rh->dictionary_values = dval;
 		}
-		else if (strncmp (buffer, "$INCLUDE", 8) == 0)
+		else if ((pfilename != NULL) && 
```

Comment:
I think that here you meant `filename != NULL` instead of `pfilename` (which is always non-NULL), right?